webapp/sources/rudder/rudder-core/src/test/resources/configuration-repository/techniques/ncf_techniques/technique_by_Rudder/1.0/metadata.xml

```diff
@@ -31,7 +31,7 @@
     <SECTION component="true" multivalued="true" name="Command execution">
       <REPORTKEYS>
         <VALUE>Write-Host &quot;testing special characters ` è &amp;é 'à é &quot;</VALUE>
-        <VALUE>/bin/echo &quot;testing special characters ` è &amp;é 'à é &quot;</VALUE>
+        <VALUE>/bin/echo &quot;testing special characters ` è &amp;é 'à é &quot;\</VALUE>
```

Comment:
why '\' ?

Reply:
Adding support of \ at the end of a value ! not breaking generated promises anymore